Release 1.18.0-RC3

xprazak2 opened the pull request:
For plugin updates, please indicate which repos this should be built into:

* [ ] Nightly
* [x] 1.18
* [ ] 1.17
* [ ] 1.16

See Foreman's [plugin maintainer documentation](https://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

ehelms commented:
I think the `foreman.spec` needs to be updated here to include a Requires on nodejs-react-debounce-input for the assets package. I tried to test build Katello and got:

```
resolve 'react-debounce-input' in '/builddir/build/BUILD/katello-3.7.0.rc1/usr/share/foreman/webpack/assets/javascripts/react_app/components/common/SearchInput'
  Parsed request is a module
  using description file: /builddir/build/BUILD/katello-3.7.0.rc1/package.json (relative path: ./usr/share/foreman/webpack/assets/javascripts/react_app/components/common/SearchInput)
    Field 'browser' doesn't contain a valid alias configuration
  after using description file: /builddir/build/BUILD/katello-3.7.0.rc1/package.json (relative path: ./usr/share/foreman/webpack/assets/javascripts/react_app/components/common/SearchInput)
    resolve as module
```

xprazak2 commented:
Updated, I added Requires for react-debounce-input, BuildRequires already present:
https://github.com/theforeman/foreman-packaging/blob/rpm/1.18/packages/foreman/foreman/foreman.spec#L387-L388
